From b94ecddd31d6d3d4db789f3da6943ac48b5dcd39 Mon Sep 17 00:00:00 2001
From: "David.Wu" <david.wu@rock-chips.com>
Date: Mon, 24 Dec 2018 19:40:52 +0800
Subject: [PATCH] pinctrl: rockchip: Add gpio3b4 io function recalculated
 select for RK3308B

Accroding to the datasheet, the pin of gpio3b4 needs to be
recalculated for iomux selecting.

Change-Id: I62cc16cae96fe2f9624d9c5940f9c34b304eaed7
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 drivers/pinctrl/pinctrl-rockchip.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/pinctrl/pinctrl-rockchip.c b/drivers/pinctrl/pinctrl-rockchip.c
index a6b2511cc67a..523d72d3073f 100644
--- a/drivers/pinctrl/pinctrl-rockchip.c
+++ b/drivers/pinctrl/pinctrl-rockchip.c
@@ -840,6 +840,12 @@ static struct rockchip_mux_recalced_data rk3308b_mux_recalced_data[] = {
 		.reg = 0x34,
 		.bit = 8,
 		.mask = 0xf
+	}, {
+		.num = 3,
+		.pin = 12,
+		.reg = 0x68,
+		.bit = 8,
+		.mask = 0xf
 	}, {
 		.num = 3,
 		.pin = 13,
-- 
2.35.3

